lib/main.dart

```diff
@@ -170,6 +170,14 @@ void main() async {
               Provider<ContentProvider>.value(
                 value: ContentProvider(client: client),
               ),
+              FutureProvider(
+                initialData: ContentProvider(client: client),
+                create: (context) async {
+                  final provider = ContentProvider(client: client);
+                  await provider.findAllOptionsByType();
```

Comment:
Looks good. I had to re-create the `Option`s, in particular `Language`s, in the remote server in order for this to work. The remote server was throwing an error because it had created these objects and stored them in MongoDB prior to the addition of `isUiLanguage`.